config/brakeman.ignore

```diff
@@ -140,6 +171,37 @@
       "confidence": "Medium",
       "note": ""
     },
+    {
+      "warning_type": "Cross-Site Scripting",
+      "warning_code": 2,
+      "fingerprint": "9809c8ef8339317eed4e032bd3c2ff36a9f6637d75f8e5d0101fde7ea2b815f2",
+      "check_name": "CrossSiteScripting",
+      "message": "Unescaped parameter value",
+      "file": "app/views/cases/show.html.slim",
+      "line": 46,
+      "link": "https://brakemanscanner.org/docs/warning_types/cross_site_scripting",
+      "code": "action_buttons_for_allowed_events(NewOverturnedIcoCaseService.new(params[:id]).original_ico_appeal.decorate, :extend_for_pit, :remove_pit_extension).join(\" \")",
+      "render_path": [
+        {
+          "type": "controller",
+          "class": "Cases::OverturnedIcoFoiController",
+          "method": "new_overturned_ico_for",
+          "line": 12,
+          "file": "app/controllers/concerns/overturned_case.rb",
+          "rendered": {
+            "name": "cases/show",
+            "file": "app/views/cases/show.html.slim"
+          }
+        }
+      ],
+      "location": {
+        "type": "template",
+        "template": "cases/show"
+      },
+      "user_input": "params[:id]",
+      "confidence": "Weak",
+      "note": "False positive - vals passed aren't from user input"
```

Comment:
just minor comment to the note here,  IMO, we can mark this as false positive, it is because we will use it params[:id] to find the case first which through ActiveRecord.find() --> which prevent any potential bad scripting. 

Based on my understanding, anything in params is sort of 'user input' to backend, no matter which way, doesn't have to be inputed via user typing

Reply:
Yep, I think the same